data/pyridine2.yml

```diff
@@ -0,0 +1,21 @@
+# pyridine
```

Comment:
I suggest deleting this file. The first pyridine.yml should be of sufficient quality.